guides/06_Meeting_Events.md

```diff
@@ -200,10 +212,14 @@ The following table shows common error messages you may receive when failing to
 
 |Messages|Status code|Suggested resolution
 |--|--|--
+|The attendee couldn't join because the meeting was at capacity.|AudioCallAtCapacity|Ensure that the number of attendees does not exceed the maximum limit (250). See the [Amazon Chime SDK quotas](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). You can also track the number of attendees in your server application using the [Amazon Chime SDK event notifications](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-notifications.html).
 |The meeting already ended.|MeetingEnded|Ensure that you or someone else have not deleted a meeting using the [DeleteMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_DeleteMeeting.html) API action in your server application. A meeting also automatically ends after a period of inactivity. See the [Chime SDK developer guide](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html) for details.
+|The signaling connection was closed with code (close code) and reason: (reason)|SignalingBadRequest|Ensure that you do not use the deleted attendee's response in the `DefaultMeetingSession` object to start the meeting. You also should not use the attendee response from the finished meeting that you created with the same `ClientRequestToken` parameter before.
```

Comment:
Should we say "ended" meeting than "finished" meeting?

Reply:
Thanks for the suggestion! I wasn't sure between "ended" and "finished" too.

Reply:
cool, thanks for updating!